<launch>


    

    <node name="tracker1" pkg="traj_tracker" type="tracker.py" output ="screen" > 
        <param name="pose_x" value="1.0" type="double" />
        <param name="pose_y" value="1.0" type="double" />
        <param name="pose_theta" value="1.57" type="double" />
        <param name="turtle" value="turtle1" />
        <rosparam param="trajectory_description" command="load" file="$(find traj_tracker)/config/N_traj.yaml" />  
        </node>

    



</launch>